Buildings/Controls/OBC/ASHRAE/G36_PR1/Generic/SetPoints/Validation/OperationMode.mo

```diff
@@ -114,120 +65,67 @@ protected
-  connect(greEqu2.y, opeModSel.unoHeaHigMin) annotation (Line(points={{62,-160},
-          {70,-160},{70,-140},{100,-140},{100,6},{128,6}}, color={255,0,255}));
-  connect(addPar.y, opeModSel.TZonMax) annotation (Line(points={{2,-70},{82,-70},
-          {82,4},{128,4}}, color={0,0,127}));
-  connect(addPar.y, opeModSel.TZonMin) annotation (Line(points={{2,-70},{82,-70},
-          {82,2},{128,2}}, color={0,0,127}));
-  connect(addPar.y, greEqu3.u1) annotation (Line(points={{2,-70},{20,-70},{20,-200},
-          {38,-200}}, color={0,0,127}));
-  connect(TZonCooSetUno.y, greEqu3.u2) annotation (Line(points={{-38,-200},{0,-200},
-          {0,-208},{38,-208}}, color={0,0,127}));
+    annotation (Line(points={{22,-80},{38,-80}},   color={255,0,255}));
+  connect(booToInt.y, opeModSel.totColZon) annotation (Line(points={{62,-80},{68,
+          -80},{68,88},{88,88}},       color={255,127,0}));
+  connect(greEqu2.y, opeModSel.unoHeaHigMin) annotation (Line(points={{22,-80},{
+          30,-80},{30,-60},{60,-60},{60,86},{88,86}},      color={255,0,255}));
+  connect(addPar.y, opeModSel.TZonMax) annotation (Line(points={{-38,10},{42,10},
+          {42,84},{88,84}},color={0,0,127}));
+  connect(addPar.y, opeModSel.TZonMin) annotation (Line(points={{-38,10},{42,10},
+          {42,82},{88,82}},color={0,0,127}));
+  connect(addPar.y, greEqu3.u1) annotation (Line(points={{-38,10},{-20,10},{-20,
+          -120},{-2,-120}},
+                      color={0,0,127}));
+  connect(TZonCooSetUno.y, greEqu3.u2) annotation (Line(points={{-78,-120},{-40,
+          -120},{-40,-128},{-2,-128}},
+                               color={0,0,127}));
   connect(greEqu3.y, booToInt1.u)
-    annotation (Line(points={{62,-200},{78,-200}}, color={255,0,255}));
-  connect(greEqu3.y, opeModSel.maxHigUnoCoo) annotation (Line(points={{62,-200},
-          {70,-200},{70,-180},{120,-180},{120,-2},{128,-2}}, color={255,0,255}));
-  connect(booToInt1.y, opeModSel.totHotZon) annotation (Line(points={{102,-200},
-          {114,-200},{114,0},{128,0}}, color={255,127,0}));
+    annotation (Line(points={{22,-120},{38,-120}}, color={255,0,255}));
+  connect(greEqu3.y, opeModSel.maxHigUnoCoo) annotation (Line(points={{22,-120},
+          {30,-120},{30,-100},{80,-100},{80,78},{88,78}},    color={255,0,255}));
+  connect(booToInt1.y, opeModSel.totHotZon) annotation (Line(points={{62,-120},{
+          74,-120},{74,80},{88,80}},   color={255,127,0}));
 
+  connect(occSch.occupied, opeModSel.uOcc) annotation (Line(points={{-79,144},{80,
+          144},{80,102},{88,102}},    color={255,0,255}));
+  connect(occSch.tNexOcc, opeModSel.tNexOcc) annotation (Line(points={{-79,156},
+          {-40,156},{-40,114},{60,114},{60,100},{88,100}},
+                                                     color={0,0,127}));
 annotation (
-  experiment(StopTime=86400, Tolerance=1e-6),
+  experiment(
+      StopTime=172800,
+      Tolerance=1e-06,
+      __Dymola_Algorithm="Dassl"),
```

Comment:
remove this `__Dymola_Algorithm="Dassl"`. It might be automatically added by Dymola 2020x.

Reply:
Removed